### PR TITLE
Feat/add redux

### PR DESCRIPTION
**Pull request by taiena**
added redux, reducer with logic of login and register user, added login page and register page with validation forms

### CODE REVIEW

File: src/redux/authReducer.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    })
+}
+
+export const register =
+  (name, email, password, passwordConfirmation, type) => async (dispatch) => {
+    await Api.register(name, email, password, passwordConfirmation, type)
+      .then((response) => {
+        if (response.success === true) {
+          // eslint-disable-next-line no-console
+          console.log('register ok')
```
Comment (alxekb) [inline src/redux/authReducer.js:98]: console.log(response.body)

File: src/const/const.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+export const getAccessToken = () => {
+  const token = localStorage.getItem('token')
```
Comment (alxekb) [inline src/const/const.js:4]: const token = localStorage.token

Comment (alxekb) [reply src/const/const.js:4]: export const getAccessToken = () => {
   return localStorage.token
}

**Review by alxekb — APPROVED** [review]

File: src/pages/Login/index.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      <div className={classes.FieldTitle}>Enter your password: </div>
+      {createField('Password', 'password', [required, maxLength30], Input, {
+        type: 'password'
+      })}
+    </div>
+
+    {error && <div className={styles.formSummaryError}>{error}</div>}
+
+    <Button
+      onClick={handleSubmit}
```
Comment (alxekb) [inline src/pages/Login/index.js:34]: onClick={(arg) => handleSubmit}

Comment (alxekb) [reply src/pages/Login/index.js:34]: @taiena

File: src/pages/Login/index.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+      <div className={classes.FieldTitle}>Enter your password: </div>
+      {createField('Password', 'password', [required, maxLength30], Input, {
+        type: 'password'
+      })}
+    </div>
+
+    {error && <div className={styles.formSummaryError}>{error}</div>}
+
+    <Button
+      onClick={(customer) => handleSubmit(customer)}
```
Comment (alxekb) [inline src/pages/Login/index.js:34]:
```
onClick={() => handleSubmit(e, 'customer')}
```

**Review by alxekb — APPROVED** [review]
👍🏻

**Review by amveng — APPROVED** [review]

**Pull request merged**